paper.md

```diff
@@ -0,0 +1,91 @@
+
+It can be overwhelming to get started and choose across all the available models, losses, and parameters. `ivadomed`'s repository includes the script `ivadomed_automate_training` to configure and launch multiple trainings across GPUs. In addition, `ivadomed` includes tutorials that cover typical training scenarios such as one-class segmentation, cascade of architectures, and uncertainty estimation. 
+
+## Evaluation
+
+A model can be thoroughly evaluated on the testing set by computing various popular metrics for segmentation, classification, and regression tasks. Slice-wise or patch-wise predictions are reconstructed in the initial space for evaluation and output visualization. `ivadomed` can produce aleatoric [@wang_aleatoric_2019] and/or epistemic [@nair_exploring_2018] uncertainty, voxel-wise and/or object-wise [@roy_quicknat_2018], using multiple available measures (e.g., entropy, coefficient of variation). Results are reported in a CSV file. The evaluation framework can be further customized with post-processing (e.g., fill holes, remove small objects, thresholding using uncertainty). It is also possible to compute metrics for specific object sizes (e.g., small vs. large lesions). `ivadomed` has a module to find the optimal threshold value on the output soft prediction, via a grid-search finding applied to evaluation metrics or ROC curve.
+
+Multiple visualization tools are included in `ivadomed` to support the design and optimization of tailored training models: GIF animations across training epochs, visual quality control of data augmentation, training curve plots, integration of the TensorBoard module, and output images with true/false positive labels. 
+
+# Usage
+
+Past or ongoing research projects using `ivadomed` are listed [here](https://github.com/ivadomed/ivadomed/docs/source/use_cases.rst).
```

Comment:
i think it would be *really* cool to add Andréanne's figure about tumor segmentation. People *love* figures.

Reply:
Yeah!! Done in [608a24d](https://github.com/ivadomed/ivadomed/pull/502/commits/608a24d9044de12cab84eb12acb37be4dacdcb5d).